hive-metastore/src/main/java/org/apache/iceberg/hive/HiveOperationsBase.java

```diff
@@ -62,6 +82,48 @@ interface HiveOperationsBase {
 
   String table();
 
+  String catalogName();
```

Comment:
I don't think we should be introducing `catalogName()`/ `contentType()` to this API just for printing it in a `TRACE` log inside `loadHmsTable()`. Please remove `catalogName()`/ `contentType()` from this API